### ISSUE TITLE
expandSubcollections breaks on 13.1

### ISSUE THREAD

**Issue by michha** [joel74/POSH-LTM-Rest#199]
In September last year there was a commit https://github.com/joel74/POSH-LTM-Rest/commit/7b0b0de1365e545d9fc92edd780fd7cc82a26313 which added a `expandSubcollections` parameter to the `Get-VirtualServer` command. This seems to be not compatible with our BIG-IP 13.1.1 as it says this parameter is unknown.

**Comment by joel74** [comment]
Hi, Thanks for noting this. I'm trying to find out is something changed with v13.1.1 but haven't been able to find anything thus far. There is a [TMOS fix](https://techdocs.f5.com/kb/en-us/products/big-ip_ltm/releasenotes/related/relnote-supplement-bigip-13-1-1-4.html) in 13.1.1.4 that references expandSubcollections in the virtual servers objects, so it appears it is a valid param there. Can you please provide some more details on the code you're executing and the full error being returned? Thanks.

**Comment by michha** [comment]
A call to `Test-VirtualServer` with a given name is successful because it doesn't append `expandSubcollections`. Otherwise when calling `Remove-VirtualServer` with a `Name` parameter it calls `Get-VirtualServer` which fails

```
F5-LTM Call: Remove-VirtualServer -Name web.myapp_vs -Partition customPartition -Confirm:$false
[Y] Yes  [N] No  [S] Suspend  [?] Help (default is "Y"):

Invoke-F5RestMethod : "400 Bad Request: Query parameter expandSubcollections is invalid.
At C:\Modules\f5-ltm\1.4.292\Public\Remove-VirtualServer.ps1:35 char:25
+ ...             Invoke-F5RestMethod -Method DELETE -Uri $URI -F5Session $ ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Invoke-F5RestMethod
```

**Comment by michha** [comment]
Any updates on this?

**Comment by joel74** [comment]
Still adjusting to life during a pandemic. I hope to do some more troubleshooting in the next couple days.

**Comment by joel74** [comment]
Hi, I was able to test this, and I repro'd the issue on both 11.6 and 14.0. I don't thin that the call to Remove-VirtualServer accepts the expandSubcollections param in any version, so I'm updating the URI called for the delete to strip it out.

**Comment by joel74** [comment]
Fixed with #200
